src/main/kotlin/app/revanced/patcher/extensions/Extensions.kt

```diff
@@ -1,5 +1,13 @@
 package app.revanced.patcher.extensions
 
 import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.builder.BuilderInstruction
+import org.jf.dexlib2.builder.MutableMethodImplementation
 
-infix fun AccessFlags.or(other: AccessFlags) = this.value or other.value
\ No newline at end of file
+infix fun AccessFlags.or(other: AccessFlags) = this.value or other.value
+
+fun MutableMethodImplementation.addInstructions(index: Int, instructions: List<BuilderInstruction>) {
```

Comment:
You could make `instructions` a `vararg` instead.

Reply:
This would prevent the following:
![image](https://user-images.githubusercontent.com/13122796/162212597-ddd495e9-a881-408e-be22-15b23de9e516.png)

Reply:
You can fix that by putting `*` in front. (unpacking the array)

Reply:
Using the spread operator would only work if we would have an array instead of a list
![image](https://user-images.githubusercontent.com/13122796/162231955-48b96088-1e2d-4059-9c1b-bdd8f1766eb6.png)